### DISCUSSION TITLE
Ifcsverchok error

### DISCUSSION THREAD

**Discussion by Alx-88** [IfcOpenShell/IfcOpenShell#3376] [General]
Hi everyone,
I was trying to activate ifcsverchok following the guide but I get this error:
![2023-07-04_08-48](https://github.com/IfcOpenShell/IfcOpenShell/assets/98451435/b52a948e-bfd4-47a7-9b32-069d10ae29a0)
Any idea why that happened?

**Comment by Moult** [comment]
This has already been fixed but we haven't yet released an update. Just did: https://github.com/IfcOpenShell/IfcOpenShell/commit/3ba7ecdb95aea8cf1cd6de9a997e5a8b4ccb9ce2
